eng/pipelines/aggregate-reports.yml

```
@@ -47,30 +47,49 @@ stages:
             publishJUnitResults: false
             goals: 'clean install'
 
+        - pwsh: |
+            New-Item eng/bomgenerator/inputdir -ItemType directory
+            Copy-Item sdk/boms/azure-sdk-bom/pom.xml eng/bomgenerator/inputdir/
+            Copy-Item eng/versioning/version_client.txt eng/bomgenerator/inputdir/
+          displayName: Copy input files for BOM generation
+
+        - task: Maven@3
+          condition: succeededOrFailed()
+          displayName: 'Generate BOM Reports'
+          inputs:
+            mavenPomFile: eng/bomgenerator/pom.xml
+            options: '$(DefaultOptions)'
+            mavenOptions: '$(MemoryOptions) $(LoggingOptions)'
+            javaHomeOption: 'JDKVersion'
+            jdkVersionOption: $(JavaBuildVersion)
+            jdkArchitectureOption: 'x64'
+            publishJUnitResults: false
+            goals: 'clean package'
+
+
         - task: Maven@3
-          displayName: 'Generate Reports'
+          displayName: 'Generate Aggregate JavaDocs'
           inputs:
             mavenPomFile: pom.xml
-            options: '$(DefaultOptions) -Dinclude-non-shipping-modules -Dgpg.skip -Paggregatereports'
+            options: '$(DefaultOptions) -T 2C -Dinclude-non-shipping-modules -DskipTests -Dgpg.skip -Dmaven.main.skip=true -Dmaven.test.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Djacoco.skip=true -Drevapi.skip=true -Pjavadoc'
```

nit: since the goal is a specific plugin goal the additional skips are superfluous 

```suggestion
            options: '$(DefaultOptions) -T 2C -Dinclude-non-shipping-modules -Pjavadoc'
```